hack/deploy-custom-nto.sh

```diff
@@ -14,8 +14,8 @@ WORKDIR=$(realpath "${0%/*}/..")
 # overriding the custom NTO image.  If you wish to upgrade your cluster,
 # do not use this script.
 
-ORG=${ORG:-openshift}	# At a minimum, you'll probably want to override this variable.
-TAG=${TAG:-$(git rev-parse --abbrev-ref HEAD)}  # You may need to override this if your git branch contains special characters
+ORG=${ORG:-$USER}	# At a minimum, you'll probably want to override this variable.
```

Comment:
this script is not meant to be used in CI, right?

Reply:
Hmm, that is a good question. I do not think we patch CVO in CI this way as it would defeat the purpose, but lets wait for Jirka.

Reply:
The script is not used in the CI.  I understand the motivation for this change.  The only downside I see is less readable tags and extra clutter/mess at image repository.  Happy to adjust my scripts to explicitly set the TAG so that I don't pollute my quay.io account if we really want this. 
/lgtm
/hold

Reply:
Yeah, I treat the tags as throwaways myself and don't care about the temporaries.

If we are agreed @jmencak, can I remove the hold?

Reply:
Sure.  I guess it is user's responsibility then to clean up their image registries then.
/hold cancel